ansible/roles/captain/vars/main.yml

```diff
@@ -1,5 +1,9 @@
 deploy_env: captain
 # supervisor_task_conf: "{{ service_home }}/supervisor"
+#
+# Defined in config directory:
+# softlayer_vpn_username: ...
+# softlayer_vpn_password: ...
```

Comment:
I think I went with the forgiveness route, but I don't think what I did was unreasonable. One I made a new user that doesn't have as many permissions as the current user in the KeyPass (though still enough that if someone had access they could obviously just delete everything) and two, if we don't believe our servers in our rackspace vpn are secure (given our no root ssh and only public key access), then we shouldn't have our cloudant password or any of our passwords stored on machines. You can make the same argument for storing our cloudant password on a remote server in a localsettings file. I guess both are still non-ideal, but if one is insecure then I'd argue the other is also insecure